pkg/virt-controller/watch/vm_test.go

```diff
@@ -5316,6 +5379,309 @@ var _ = Describe("VirtualMachine", func() {
 				})
 			})
 		})
+
+		Context("The RestartRequired condition", func() {
+			var vm *virtv1.VirtualMachine
+			var vmi *virtv1.VirtualMachineInstance
+			var kv *virtv1.KubeVirt
+			var crList appsv1.ControllerRevisionList
+			var crListLock sync.Mutex
+
+			expectRestartRequired := func(shouldExpect bool) {
+				Eventually(func() {
```

Comment:
Why is `Eventually` required? The unit tests are synchronous, once the `Execute` returns the changes should be propagated.

Reply:
That's a good question, but it definitely is required...